lib/console_web/controllers/gateway_controller.ex

```diff
@@ -61,6 +62,29 @@ defmodule ConsoleWeb.GatewayController do
     end
   end
 
+  def register(conn, %{"OUI" => oui, "nonce" => nonce, "gateway" => %{"id" => gateway_id, "public_key" => public_key, "payee_address" => payee_address}}) do
```

Comment:
this should be within the router gateway controller, not the user-facing one

Reply:
wow I'm blind... thanks